### DISCUSSION TITLE
Unable to bridge

### DISCUSSION THREAD

**Discussion by sirvits10** [community/community#108261] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I have been trying several times to complete my beta testing to bridge my venome account to a EVM able app like metemask to no avail. The time to complete the transaction always EXPIRED.  Plz, fix this bug...it's too annoying. Thanks.

**Comment by VerzatileDev** [comment]
Could you provide more context?

About the steps you took,

What type of research you did and some context about the system you are using as for example.
